packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import EmergencyBanner from './EmergencyBanner';
+
+export default {
+  title: 'Library/EmergencyBanner',
+  component: EmergencyBanner,
+  argTypes: {
+    visible: { control: 'boolean' },
+    title: { control: 'text' },
+    content: { control: 'text' },
+  },
```

Comment:
All these are defined in the component's `propTypes`, so I don't think you need these.

Reply:
updated to show types instead tks